### DISCUSSION TITLE
Connect to Payson

### DISCUSSION THREAD

**Discussion by kristianbendren** [juspay/hyperswitch#3571] [Ideas / Feature Requests]
Connect to Payson payment provider.

https://www.payson.se/en/
https://tech.payson.se/paysoncheckout2/rest-api/

This is one of the largest payment providers in Sweden

**Comment by VedantKhairnar** [comment]
Hey @kristianbendren ,
Thanks for the suggestion and sharing the API docs.

Had a few questions:
- Are you requesting this new connector as a user of HyperSwitch or as a representative of Payson? Wanted to know your motivation behind the suggestion.
- Will you be interested in contributing to HyperSwitch by adding the connector and submitting a pull request?

Thanks.  : )

**Reply by kristianbendren** [reply]
As a user of HyperSwitch, or potential user of hyperswitch really, as we are today having integration against them for our booking tool and need to have this support to be able to start using Hyperswitch

Sorry dont have the time for this as we want to migrate from our own integration to an existing one.
